l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -188,20 +189,15 @@ def invoiceCreate(self, cr, uid,
             'payment_term': False,
             'company_id': company.id,
             'fatturapa_attachment_in_id': fatturapa_attachment.id,
-            'comment': comment,
+            'comment':
+            FatturaBody.DatiGenerali.DatiGeneraliDocumento.Causale[0],
```

Comment:
si si vero

Il giorno 16 marzo 2015 15:33, Lorenzo Battistini <notifications@github.com>
ha scritto:

> In l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py
> <https://github.com/eLBati/l10n-italy/pull/5#discussion_r26488771>:
>
> > @@ -188,20 +189,15 @@ def invoiceCreate(self, cr, uid,
> >              'payment_term': False,
> >              'company_id': company.id,
> >              'fatturapa_attachment_in_id': fatturapa_attachment.id,
> > -            'comment': comment,
> > +            'comment':
> > +            FatturaBody.DatiGenerali.DatiGeneraliDocumento.Causale[0],
>
> minOccurs="0" maxOccurs="unbounded" vuole dire che ce ne possono essere
> da 0 a N.
> Ti consiglio di fare riferimento al PDF tabellare che mi sembra più
> leggibile. Li relativamente a 2.1.1.11 <Causale> puoi vedere <0.N> che
> vuole dire appunto che ce ne possono essere da 0 a N.
> Quindi bisogna ciclare su tutti
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/eLBati/l10n-italy/pull/5/files#r26488771>.
>



-- 
Best Regards
Alessio Gerace


*   Think Studio*
     +39 3396911693
     www.thinkstudio.it